geode-core/src/main/java/org/apache/geode/distributed/ServerLauncher.java

```diff
@@ -169,7 +172,7 @@
   static {
     Map<Command, String> usage = new TreeMap<>();
     usage.put(Command.START,
-        "start <member-name> [--assign-buckets] [--disable-default-server] [--rebalance] [--server-bind-address=<IP-address>] [--server-port=<port>] [--force] [--debug] [--help]");
+        "start <member-name> [--assign-buckets] [--disable-default-server] [--rebalance] [--server-bind-address=<IP-address>] [--server-port=<port>] [--force] [--membership-bind-address=<IP-address>] [--debug] [--help]");
```

Comment:
For easier reading, please position the new `--membership-bind-address` param before `--force`.